semantic-conventions/src/tests/data/compat/enum_value_removed/vprev.yaml

```diff
@@ -0,0 +1,17 @@
+groups:
+  - id: first
+    type: attribute_group
+    brief: "first group"
+    note: "first group note"
+    prefix: "first"
+    attributes:
+      - id: third_attr
+        type:
+          members:
+            - id: enum_one
+              brief: "enum one"
+              value: "one"
```

Comment:
This example is the same as `enum_type_changed`. Perhaps we want to in `vnext` rename the attribute entirely so it really shows it was removed?

Reply:
`enum_type_changed` tests `"one"` -> `1`, i.e. string to int
This check is redundant (since we check that value remains the same), but I'd still prefer to keep it for the sake of better error message.

But your comment made me realize that I mistakenly allowed member id to change (as long as value remains) - this should not be allowed since codegen would do `enum Foo {  {id} = {value} }` and both should remain intact.

I renamed this test case to `value_changed` and added one more for `member_removed`

Thanks!